lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -121,13 +123,30 @@ def #{configuration[:column]}=(position)
 
             after_commit 'remove_instance_variable(:@scope_changed) if defined?(@scope_changed)'
 
-            scope :in_list, lambda { where(%q{#{quoted_table_name}.#{connection.quote_column_name(configuration[:column])} IS NOT NULL}) }
+            scope :in_list, lambda { where(%q{#{quoted_table_name}.#{quoted_position_column} IS NOT NULL}) }
+
+            def self.decrement_all_with_touch
+              update_all_with_touch %q(#{quoted_position_column} = (#{quoted_position_column} - 1))
+            end
+
+            def self.increment_all_with_touch
+              update_all_with_touch %q(#{quoted_position_column} = (#{quoted_position_column} + 1))
+            end
           EOV
 
           if configuration[:add_new_at].present?
             self.send :before_create, :"add_to_list_#{configuration[:add_new_at]}"
           end
+        end
 
+        private
+
+        def update_all_with_touch(updates)
+          attrs = new.send(:timestamp_attributes_for_update_in_model)
+          query = attrs.map { |attr| "#{attr} = :now" }
+          query.push updates
```

Comment:
Did a quick spike to see what it would look like, if you're curious. Would definitely need to be extracted to its own module so that it could be properly tested. The `when Hash` clause is dead, untested code, given the method's current usage.
```ruby
def update_all_with_touch(updates)
  record = new
  attrs = record.send(:timestamp_attributes_for_update_in_model)
  now = record.send(:current_time_from_proper_timezone)

  case updates
  when String, Array
    query = attrs.map { |attr| "#{attr} = :now" }
    query.push Array(updates).first
    query = query.join(", ")

    values = Array(updates).second || {}
    values[:now] = now

    update_all([query, values])

  when Hash
    query = attrs.inject(updates) do |hash, attr|
      hash.merge attr => now
    end
    update_all(query)

  else
    raise ArgumentError
  end
end
```